gravitee-apim-portal-webui-next/src/components/api-filter/api-filter.component.html

```diff
@@ -0,0 +1,38 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-button-toggle-group class="api-filter" [formControl]="toggleControl" [value]="'all'" (change)="onSelectedFilter($event.value)">
+  <mat-button-toggle [value]="'all'" class="api-filter__button">All</mat-button-toggle>
+  @for (filter of filterList | slice: 0 : 4; track filter.id) {
+    <mat-button-toggle [value]="filter.id" class="api-filter__button">
+      {{ filter.name }}
+    </mat-button-toggle>
+  }
+  <mat-button-toggle class="api-filter__button" [checked]="!!selectedOption" (click)="openFilterSelect()">
+    {{ selectedOption }}
+    <mat-icon>expand_more</mat-icon>
+  </mat-button-toggle>
+</mat-button-toggle-group>
+@if (isFilterOpen) {
+  <div class="api-filter-container">
```

Comment:
What about using `mat-menu` here instead of a hand-made drop-down? I used `mat-menu` for the user avatar and it matches the Figma coloring/formatting.

Reply:
In any case, the drop-down to the right is a bit awkward. It either needs be aligned manually, or `mat-menu` will align it like in the Figma. 
![Screenshot 2024-05-13 at 11 26 17](https://github.com/gravitee-io/gravitee-api-management/assets/42294616/a63d4486-13ad-44ab-8104-2d0e68e2612b)

Reply:
Also, the full name of the category should be shown when selected? In the picture, it shows only `Test Categ` instead of `Test Category`